modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1246,8 +1246,10 @@ To continue using it, escape the identifier by adding backticks around the ident
 To continue using it, escape the identifier by adding backticks around the identifier `%s`.
 - label:new[Valid for Neo4j 5.5-5.25] All subqueries in a UNION [ALL] should have the same ordering for the return columns.
 Using differently ordered return items in a UNION [ALL] clause is deprecated and will be removed in a future version.
-- Databases and aliases with unescaped `.` are deprecated unless to indicate that they belong to a composite database.
+- label:new[Valid for Neo4j 5.26-2025.05] Databases and aliases with unescaped `.` are deprecated unless to indicate that they belong to a composite database.
```

Comment:
I reused the format of the label from the entry above, but that is not visible in the published docs of some reason